app/src/main/java/com/saucelabs/mydemoapp/android/view/activities/MainActivity.java

```diff
@@ -316,32 +292,32 @@ private void handleMenuClick(int position) {
 				showResetDialog();
 				break;
 			case 7:
-//                setFragment(FRAGMENT_BIOMETRICS, param1, param2, param3);
-				if (currentFragment instanceof BiometricFragment) {
-
-				} else {
+				if (!(currentFragment instanceof BiometricFragment)) {
 					ST.startMainActivity(mAct, ST.getBundle(FRAGMENT_BIOMETRICS, 1));
 				}
 
 				break;
 			case 8:
+				startActivity(new Intent(this, VirtualUsbActivity.class));
+				break;
+			case 9:
 				TestFairy.setFeedbackOptions(new FeedbackOptions.Builder().build());
 				TestFairy.showFeedbackForm();
 				break;
-			case 9:
+			case 10:
 				startActivity(new Intent(this, DebugFeedbackActivity.class));
 				break;
-			case 10:
+			case 11:
 				startActivity(new Intent(this, DebugCrashActivity.class));
 				break;
-			case 11:
+			case 12:
```

Comment:
This change for login will break some tests here: https://github.com/saucelabs/my-demo-app-android/blob/main/app/src/androidTest/java/com/saucelabs/mydemoapp/android/view/activities/LoginTest.java

Number 11 is also hardcoded in the tests.